sky/templates/kubernetes-ray.yml.j2

```diff
@@ -126,6 +125,7 @@ available_node_types:
             parent: skypilot
             component: {{cluster_name}}-ray-head
             skypilot-cluster: {{cluster_name}}
+            skypilot-sshjump: {{sshjump}}
```

Comment:
What do you think about adding a comment on this to say that the label is used to manage the life cycle of the ssh jump pod?